context/brand-context/nature/scss/00-tokens/_illustration.variables.scss

```diff
@@ -0,0 +1,9 @@
+
+// Do not edit directly
+// Generated on Wed, 27 Apr 2022 14:51:22 GMT
+
+$illustration-background-001: #29303C;
+$illustration-background-002: {nature-color.gunmetal.500;
```

Comment:
another stray `{`, not sure about the rest?

Reply:
I've got this fixed locally now - will push a new commit in a bit

Reply:
fixed in [this commit](https://github.com/springernature/frontend-toolkits/pull/735/commits/c7cdb34c3e9db545055ae6c6577c19a57f563732) and [this other commit](https://github.com/springernature/frontend-toolkits/pull/735/commits/49a0d698182ac67bbec4b892f4cc8f21883a04da)